dbscheme-proposal.md

```diff
@@ -0,0 +1,120 @@
+# Database Scheme
+
+## scheme
+* type (enc/sig)
+* name
+* category (lattice etc)
+* exact problems plus type of reduction (list of tuples?)
+* year (what date to use?)
+* inventors ?
```

Comment:
I think I'd do something like original publication for year and inventors?

Reply:
And in addition having the NIST submission (or other standards submission/rfc) in here if there's one would be nice.